app/assets/stylesheets/application.css.scss

```diff
@@ -124,16 +132,33 @@ input {
   border-color: white;
   border: 1px;
   border-radius: 4px;
+  height: 75%;
+  margin-top: 2%;
+  margin-right: 2%;
+  margin: 0 auto;
 }
 
 #results:empty {
   display: none;
 }
 
-.image-container > img {
+#content > img {
   // max-height: 200px;
-  height: 100%;
-  width: 100%;
+  width: 80%;
+  margin: 0 auto;
+  margin-top: 4%;
+  border-radius: 4px;
+  border: 1px;
+  border-style: solid; 
+  border-color: #9ECAE1;
```

Comment:
Color literals like `#9ECAE1` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Color `#9ECAE1` should be written as `#9ecae1`